query/src/main/scala/hmda/query/repository/filing/FilingComponent.scala

```diff
@@ -0,0 +1,291 @@
+package hmda.query.repository.filing
+
+import hmda.query.DbConfiguration
+import hmda.query.model.filing.LoanApplicationRegisterQuery
+import hmda.query.repository.Repository
+import slick.basic.DatabaseConfig
+import slick.jdbc.JdbcProfile
+import slick.collection.heterogeneous._
+import slick.collection.heterogeneous.syntax._
+
+trait FilingComponent { this: DbConfiguration =>
```

Comment:
This PR doesn't deal with TS, but yes, naming it this way is because we will probably want to also aggregate TS data. In that case I would include the relevant classes / traits in this component